src/index.ts

```diff
@@ -4,12 +4,22 @@ const reserved = /^(?:do|if|in|for|int|let|new|try|var|byte|case|char|else|enum|
 const unsafe = /[<>\/\u2028\u2029]/g;
 const escaped: Record<string, string> = { '<': '\\u003C', '>' : '\\u003E', '/': '\\u002F', '\u2028': '\\u2028', '\u2029': '\\u2029' };
 const objectProtoOwnPropertyNames = Object.getOwnPropertyNames(Object.prototype).sort().join('\0');
+const defaultLogLevel = process.env.NUXT_ENV_DEVALUE_LOG_LEVEL || 'warn';
```

Comment:
Suggest adding a link (comment) to the line above to original issue. When someone reads source code can quickly find the story behind this ENV.